### PR TITLE
Add new config to allow disabling dragging for graph

### PR DESCRIPTION
**Pull request by cyyyu**
Also released @0.1.7

### CODE REVIEW

File: src/components/graph/Graph.types.ts [hunk 1]
```diff
@@ -28,6 +28,7 @@ export interface IGraphConfigZoom {
   maxZoom: number;
   minZoom: number;
   initialZoom: number;
+  dragGraphEnabled?: boolean;
```
Comment (warmsea) [inline src/components/graph/Graph.types.ts]: To be consistent with `zoomByScroll` could you rename it `panByDrag`?

File: stories/graph/Behavior.stories.tsx [hunk 2]
```diff
@@ -1,15 +1,22 @@
...
+import { IGraphProps } from "../../src/components/graph/Graph.types";
 
 const meta: Meta = {
   title: "Graph: Behavior"
 };
 
 export default meta;
 
-export const Zoom: FC = () => {
+interface ITemplateArgs extends BaseStory<Args, StoryFnReactReturnType> {
```
Comment (warmsea) [inline stories/graph/Behavior.stories.tsx]: Please don't add so many items. Storybook supports you to update parameters on its UI. I'm also OK if you don't update  the story in this PR.

File: src/components/graph/Graph.types.ts (hunk 1 above)
Comment (cyyyu) [reply src/components/graph/Graph.types.ts]: thanks. updated.

File: src/components/graph/Graph.types.ts [hunk 3]
```diff
@@ -28,6 +28,7 @@ export interface IGraphConfigZoom {
   maxZoom: number;
   minZoom: number;
   initialZoom: number;
+  panByDrag?: boolean;
```
Comment (warmsea) [inline src/components/graph/Graph.types.ts:31]: Please set the default value to be `true` in graph.config.ts

**Review by warmsea — APPROVED** [review]

**Pull request merged**